source/fundamentals/crud/read-operations/cursor.txt

```diff
@@ -114,6 +114,24 @@ through results rather than returning all documents at once.
    :start-after: start fetchAll cursor example
    :end-before: end fetchAll cursor example
 
+Return Search Indexes of Collection
```

Comment:
I: I'm not sure that this content belongs on this page. This page is about different ways to access data from a cursor, not about methods that return cursors. I think that you could add a section instead to the Indexes fundamentals page that covers this new method and the existing `listIndexes()` method. 

S: Move this section to Indexes and add information also about using `listIndexes()`. On this page, the only change that should be made is adding this method the list at the top of the page under "The following functions directly return cursors:"